ctapipe/core/tool.py

```diff
@@ -137,6 +137,7 @@ def initialize(self, argv=None):
             except Exception as err:
                 raise ToolConfigurationError(f"Couldn't read config file: {err}")
         self.log.info(f"ctapipe version {self.version_string}")
+        self.parse_command_line(argv)  # command-line takes precedence
```

Comment:
From traitlets code here:
https://github.com/ipython/traitlets/blob/master/traitlets/config/application.py

We see that the parsed command line options are stored in `self.cli_config`:
https://github.com/ipython/traitlets/blob/32ced29b0ab06914ff92e9bdcd8806536633d701/traitlets/config/application.py#L704-L705

Instead of parsing again, we could just call `self.update_config(self.cli_config)` again, that probably avoids the parsing step and still does what we want.

Reply:
Just re-parsing is pretty fast, and probably avoids it breaking everything if they change the internals of traitlets.config (which is annoyingly very poorly documented). But i'm ok either way.

Reply:
I like the `update_config` better (more self-explanatory imo) but with soft preference so good either-way.

Reply:
ok changed to use update_config